doc/user/download_backup.md

```diff
@@ -21,7 +21,10 @@ $ curl "http://etcd-cluster-backup-sidecar:19999/v1/backup?etcdVersion=3.1.0" -o
 On success, the backup's etcd version should be compatible with given version.
 Otherwise, it would return non-OK response.
 
-It's up to users to decide how to access the backup service, e.g. using [ingress](https://kubernetes.io/docs/user-guide/ingress/) .
+In other namespace, use FQDN `${CLUSTER_NAME}-backup-sidecar.${NAMESPACE}.svc.cluster.local` .
```

Comment:
I think he probably means sending the GET request from a pod in a namespace different from the backup service. To access services outside your namespace you need FQDN.